### ISSUE TITLE
NVDA could not completely support the input method New Phonetic 2010

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2792]
**Reported by surfer0627 on 2012-11-11 10:03**
The case description is below:
  User type Chinese words and press down arrow to select from the candidate list.

Then user press enter to select candidate.
The window stuck in the candidate list, and the functions (voice and braille display) are broken.

User need to press Alt+ tab to shift the window, then the functions could work again.

If necessary, please download the software New Phonetic 2010 from the links below. (file size: 131mb)
https://dl.dropbox.com/u/90288447/IME2010_zh-tw.exe

Thank you.

**Comment by bhavyashah** [comment]
@surfer0627 Does this still occur for you? If so, is this issue specific to the Chinese input tool you specified, i.e. is this app specific, or does this happen with other input methods also? In case it is the former, please test with the latest version of the software in question.

**Comment by Adriani90** [comment]
@surfer0627 how is this now in NVDA 2020.1 RC1? Can you still reproduce this issue?

**Comment by Adriani90** [comment]
@surfer0627 do you have any updates on how it works with nVDA 2022.4 or 2023.1 Beta? Otherwise we have to close this as abandoned due to missing updates.

**Comment by Adriani90** [comment]
cc: @larry801 can you test as well?

**Comment by Adriani90** [comment]
I am closing this issue as abandoned, no updates from the initial author and the link for the test case doesn't work anymore. If you are still having this issue, please comment and we can reopen.
cc: @cary-rowen fyi

**Comment by cary-rowen** [comment]
Yes, I agree with closing this, and if the original author wishes to reopen it, I recommend re-reporting the issue using the latest Issue template.
